papas/datatypes/Cluster.h

```diff
@@ -52,14 +63,13 @@ class Cluster {
   const std::list<const Cluster*>& subClusters() const { return m_subClusters; };
   std::string info() const;  ///< returns a text descriptor of the cluster
 
-  static double maxEnergy() {
-    return s_maxEnergy;
-  };  ///< static that returns max cluster energy (intended for display purposes)
+  /// static that returns max cluster energy (intended for display purposes)
+  static double maxEnergy() { return s_maxEnergy; };
+
 protected:
   Identifier m_id;                          ///< identifier for Cluster
   double m_size;                            ///< Cluster size (radius?)
   double m_angularSize;                     ///< Cluster angular size (only valid for non-merged clusters)
-  double m_pt;                              ///< Transverse momentum (magnitude of p3 in transverse plane)
   TVector3 m_p3;                            ///< position (x, y, z)
   double m_energy;                          ///< Energy
```

Comment:
I think this is a misunderstanding because of the misnomer of `m_p3` which sounds like momentum but indeed is the position (at least the Doxygen comment says so).